test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -938,22 +938,6 @@ class TestDapp {
     );
   }
 
-  /**
-   * Sign a message with the signTypedDataV3 method.
-   */
-  async signTypedDataV3() {
-    console.log('Sign message with signTypedDataV3');
-    await this.clickSignTypedDatav3();
-    await this.driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-    await this.driver.waitForSelector(
-      this.signTypedDataV3V4SignatureRequestMessage,
-    );
-    await this.driver.clickElementSafe(this.confirmDialogScrollButton, 200);
-    await this.driver.clickElementAndWaitForWindowToClose(
-      this.confirmSignatureButtonRedesign,
-    );
-  }
-
   async signTypedDataV3Redesign() {
```

Comment:
Tangent to the PR, but it seems to me an incorrect pattern here: in the test dapp class we are including methods and actions that should belong to Confirmations Signature class :thinking: 

I saw this was introduced [here](https://github.com/MetaMask/metamask-extension/pull/29819/files#diff-454b7295f72a1d2429b870f87faafc5aebf542ff3a8b0775ac14d87ef496118bR734).


To me mixing actions from different page objects is not desirable, rather we should create a flow, if we want to include actions for multiple page objects. What do you think? Let me know if this also make sense to you and I can raise this to the Confirmations team

Reply:
this means we shouldn't have a signTypedDataV3 and signTypedDataV3Redesign, because the test dapp doesn't "care" if we are using the redesign confirmation screen or the legacy one. So we should have just a generic name here. What do you think?

Reply:
yes! I totally agree. I think what we should do is:

- Move the signature flows to `test/e2e/page-objects/flows` and remove them from the `testDapp` class, since they involve interactions across multiple pages.
- Rename/standardize all function names that include `Redesign`. Now that the screen redesign is complete, we should standardize the names (e.g., rename `signTypedDataV3Redesign` to `signTypedDataV3`). The term `Redesign` should no longer appear in E2E tests, as it creates confusion, especially for new contributors who may not know what "redesign" refers to.
- Agree we can handle in a separate PR, as it will involve much broader changes. I've created a ticket to track this work here: https://consensyssoftware.atlassian.net/browse/MMQA-533

Reply:
make sense to me! thank you for creating the ticket :pray: